contract/bridge/contracts/OneBtc.sol

```diff
@@ -58,15 +58,20 @@ contract OneBtc is ERC20Upgradeable, Issue, Redeem, Replace {
         uint256 issueId,
         bytes calldata merkleProof,
         bytes calldata rawTx, // avoid compiler error: stack too deep
-    //bytes calldata _version, bytes calldata _vin, bytes calldata _vout, bytes calldata _locktime,
-        uint32 height,
-        uint256 index,
-        bytes calldata header
+        //bytes calldata _version, bytes calldata _vin, bytes calldata _vout, bytes calldata _locktime,
+        uint64 heightAndIndex,
```

Comment:
is this change required? `heightAndIndex` instead of separate `height` and `index` parameters?

Reply:
because i have added one more argument: outputIndex and after this i got error on build: `deep stack error` - to resolve it i have contained height & Index to common argument heightAndIndex